.github/workflows/python-lint.yml

```diff
@@ -20,5 +20,5 @@ jobs:
       - name: flake8 Lint
         uses: py-actions/flake8@v2
         with:
-            ignore: "E203,E211,E221,E225,E231,E241,E251,E261,E266,E501,E701,E704,W503"
+            ignore: "E203,E211,E221,E225,E231,E241,E251,E261,E266,E501,E701,E704,E722,W503"
```

Comment:
No I guess it had been fixed somewhere else, please rebase onto master and remove

Reply:
Ah great, I'll rebase now. Thanks